influence_benchmark/vectorized_environment/vectorized_environment.py

```diff
@@ -55,14 +60,16 @@ def replace_environment(self, env_id: int):
         if new_env is None:
             del self.environments[env_id]
             self.vectorized_preference_model.remove_model(env_id)
+            self.vectorized_influence_detector_model.remove_model(env_id)
             self.vectorized_transition_model.remove_model(env_id)
-            self.vectorized_character.remove_character(env_id)
+            self.vectorized_character.remove_model(env_id)
             del self.traj_count[env_id]
         else:
             self.environments[env_id] = new_env["environment"]
-            self.vectorized_transition_model.replace_model(new_env["transition_model"], env_id)
             self.vectorized_preference_model.replace_model(new_env["preference_model"], env_id)
-            self.vectorized_character.replace_character(new_env["character"], env_id)
+            self.vectorized_influence_detector_model.replace_model(new_env["transition_model"], env_id)
```

Comment:
don't think this should be "transition_model"

Reply:
fixed